server/covmanager/migrations/0006_auto_20210429_0908.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.0.14 on 2021-04-29 09:08
+
+import django.core.files.storage
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('covmanager', '0005_report'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='collectionfile',
+            name='file',
+            field=models.FileField(
+                max_length=255,
+                storage=django.core.files.storage.FileSystemStorage(location=None),
+                upload_to='coverage'
```

Comment:
The only change I can see from [0002_increase_collection_filename_length.py](/MozillaSecurity/FuzzManager/blob/master/server/covmanager/migrations/0002_increase_collection_filename_length.py) is `upload_to` changes from `bytes` to `str`. 👍🏻